lib/trip.rb

```diff
@@ -1,14 +1,22 @@
 require 'csv'
+require 'driver'
+require 'pry'
 
```

Comment:
The `require 'driver'` here prevented me from running this code. It needs to be `require_relative`.